pkg/vrct/vrctFs/config.go

```diff
@@ -0,0 +1,97 @@
+package vrctFs
+
+import (
+	"errors"
+	"os"
+	"path/filepath"
+)
+
+// CreateConfig function creating or updating configuration file
+//
+// Arguments:
+//
+//	filePath - Path to file
+//	content - content of file
+//	optionalKeys - json or yaml document describing which key in config is optional
+//	isOptional - default option in configs / is able to merge in text files
+//	fileType - given 0 - text file, otherwise config specified in file_type.go
+func (v *VRCTFs) CreateConfig(filePath string, content []byte, optionalKeys []byte, isOptional bool, fileType FileType) error {
+	filePath, err := filepath.Abs(filePath)
+	if err != nil {
+		return err
+	}
+	dirPath := filepath.Dir(filePath)
+
+	err = os.MkdirAll(filepath.Join(v.virtualFSPath, dirPath), os.ModePerm)
+	if err != nil {
+		return err
+	}
+
+	filePrototype := FilePrototype{
+		FileType: fileType,
+	}
+	err = filePrototype.Read(v.virtualFSPath, filePath)
+	if err != nil {
+		return err
+	}
+
+	if filePrototype.FileType == TextFile {
+		return errors.New("trying to create file, where it's config type")
```

Comment:
Could you explain this error message to me?

Reply:
When you create file `/tmp/file` with no content it is not supposed to be a configuration file. You can try to add `/tmp/file` with json content. They should not be merged